### ISSUE TITLE
Color contrast in nteract dropdown menu is not accessible

### ISSUE THREAD

**Issue by miduncan** [nteract/archived-desktop-app#4759]
Currently, the hovered item in the nteract/dropdown-menu puts a dark blue text over a darkish-gray background. 
![image](https://user-images.githubusercontent.com/55718965/70586996-9ef5eb00-1b7d-11ea-971a-9fd84f045725.png)

![image](https://user-images.githubusercontent.com/55718965/70586899-4a527000-1b7d-11ea-9e34-1564eb2ee259.png)

4.5 is the target contrast ratio (or higher). The closest gray I found that passes this is #E4E4E4

**Comment by captainsafia** [comment]
Thanks for reporting this!

I saw this in an automated accessibility check a while back eventually deemed it to low-pri to fix in context of upcoming UX changes. I'll leave it open so people on the current UI know.

**Comment by MichelleUfford** [comment]
@javag97 this one has your name written all over it! :)

**Comment by javag97** [comment]
#E2E5E7 (grey-3) looks like should work fine. Let's see what we can do about this.
